docs/tutorials/10_train_custom_env.ipynb

```diff
@@ -5,18 +5,17 @@
    "metadata": {},
```

Comment:
Newlines seem to have changed throughout this and many other notebooks making the diff messy. Can you clean this up? Most of the notebooks shouldn't be changing in this PR, and I can't tell right now which notebooks had actual semantic changes.

Reply:
Sorry about this. Yes, I did some semantic changes and then the lint keeps failing on `docs/tutorials/4_train_airl.ipynb`. I did something wrong and formatted all notebook. I should have just formatted that only one. 

I have reverted the commit and now the notebook change should be easy to read.